src/components/annotations/attributes.js

```diff
@@ -49,6 +49,27 @@ module.exports = {
     font: extendFlat({}, fontAttrs, {
         description: 'Sets the annotation text font.'
     }),
+    width: {
+        valType: 'number',
+        min: 0,
+        dflt: 0,
```

Comment:
Obviously the default value isn't `0`, it's computed from the input text. I believe in plotly.js lingo that means the `dflt` value should be set to `null` (similar to e.g. `contours.start` and `xbins.start`)